client/src/components/TagSelector/Selection.tsx

```diff
@@ -0,0 +1,36 @@
+import type { SelectionProps } from "@/components/TagSelector/tag-selector.types";
+import { makePath } from "@/lib/tag-path";
+import type { TagWithIds } from "@/types/server/tag.types";
+import { Fragment } from "react/jsx-runtime";
+import S from "./TagSelector.style";
+
+function Selection(p: SelectionProps) {
+	if (!p.selectedTags.length) return null;
+
+	function renderPath(tag: TagWithIds, tags: TagWithIds[]) {
+		return (
+			<S.SelectionItem key={tag.tag_id}>
+				{p.fullPaths
```

Comment:
Do I prefer a ternary here, or two separate checks, `!p.fullPaths &&` and `!!p.fullPaths &&`?

Reply:
Ternary is readable enough in this case.